python/lsst/ts/eas/hvac_model.py

```diff
@@ -52,6 +52,10 @@ class HvacModel:
         A model representing the dome state.
     weather_model : `WeatherModel`
         A model representing weather conditions.
+    ahu_setpoint_delta : `float`
+        The offset that will be applied to the measured temperature in
```

Comment:
applied -> added, as that better explains if there is + or -. Also in doc section below.